### PR TITLE
Adds Visual Regression Testing to best practices

### PR DESCRIPTION
**Pull request by ryansommers**
### Description of the Change

Adds Visual Regression Testing to Tools section now that it's part of the theme scaffold.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: _includes/markdown/Tools.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ We're also a fan of a few browser tools that lend us a hand when it comes to tes
 * [Contrast Ratio](https://leaverou.github.io/contrast-ratio/) - A color contrast tool to compare two colors against [levels of conformance](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html) and see if they pass.
 * [Tanagaru Contrast Finder](http://contrast-finder.tanaguru.com/?lang=en) - Another color contrast tool that tests colors against the levels of conformance, but also provides you with alternatives should your provided colors fail.
 
-<h3 id="vrt" class="anchor-heading">Visual Regression Testing</h3>
+<h2 id="vrt" class="anchor-heading">Visual Regression Testing</h2>
 
 We do visual regression testing to ensure code changes don't have unforeseen repercussions. This provides a helpful visual aid to check against CSS changes, plugin updates, and third-party script updates.
```
Comment (timwright12) [inline _includes/markdown/Tools.md]: We *use* visual

Comment (timwright12) [inline _includes/markdown/Tools.md]: Can we list some common tools and elaborate a bit more on the methodology and benefit? It'd be nice to be able to point accounts here when we need more info about VRT

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: _includes/markdown/Tools.md (hunk 1 above)
Comment (ryansommers) [reply _includes/markdown/Tools.md]: @timwright12 I was trying to stick with the format of the rest of the Tools page, which is a pretty dry list of specifically what tools we use. This doesn't seem like the right place to "sell" something – we're not elaborating here on why we do a11y testing and why it's beneficial either, for instance. 

Do we want to list other VRT tools we don't actually use here, or only the one we're adding to our scaffolding?

Comment (ryansommers) [reply _includes/markdown/Tools.md]: Fixed

Comment (timwright12) [reply _includes/markdown/Tools.md]: yeah, list the tools we use and why we use them, like the rest of the page. We probably need to redo the accessibility one, I don't know when it was written

**Review by timwright12 — APPROVED** [review]

**Pull request merged**